src/shared/domain/wizard-page.model.ts

```diff
@@ -0,0 +1,35 @@
+import { Orderable } from './order/orderable.model';
+import { WizardPageField } from './wizard-page-field.model';
+import { CaseField } from './definition/case-field.model';
+import { ShowCondition } from '../conditional-show/conditional-show.model';
+import { Type } from 'class-transformer';
+
+// @dynamic
```

Comment:
You will see these added in few places. These are to let the compiler know the arrow syntax is acceptable in this class. Otherwise the collector in the compiler was barking at me. :(